packages/pwa-devdocs/src/venia-pwa-concept/simple-rest-checkout/index.md

```diff
@@ -0,0 +1,85 @@
+the checkout flow described in this topic uses the REST API to complete the process.
+
+The following is a list of Magento REST endpoints and HTTP methods used for this checkout flow:
+
+| REST Endpoint                     | HTTP method | Usage                                                   |
+| --------------------------------- | ----------- | ------------------------------------------------------- |
+| `/V1/guest-carts`                 | POST        | Get information about or create and store a new cart ID |
+| `/V1/guest-carts/{cartId}/items`  | POST        | Update cart with item details                           |
+| `/V1/guest-carts/{cartId}`        | GET         | Get information about the cart items                    |
+| `/V1/guest-carts/{cartId}/totals` | GET         | Get information about cart totals                       |
+{:style="table-layout:auto"}
+
+For a list of Magento REST endpoints, see the [REST API documentation][] in devdocs.
+
+## Shopper experience
+
+Before going into the technical steps that happens in the background, you should be familiar with the events surrounding the checkout process.
+
+The following steps summarize the basic checkout experience for an online shopper in `Venia`:
+
+1. The shopper navigates to a product page.
+1. The shopper adds an item to the cart using the **Add to Cart** button.
+1. The shopping cart drawer slides out as a response to the add item action.
+1. The shopper clicks on the **Checkout** button.
+1. The **Shipping and Billing** form appears and replaces the shopping cart drawer.
+1. The shopper fills out the form or uses preset account information and clicks on the **Submit Order** button.
+1. A confirmation page appears and the shopper clicks on **Continue Shopping** to go back to the product page.
+
+## Detailed technical flow
+
+The following sections provide the technical details for each step in the checkout flow.
+
+### Updating the cart
+
+1. When the shopper clicks on the **Add To Cart** button, the [Product component][] calls the `addItemToCart` [action creator][] and waits for the action resolution.
+2. The `addItemToCart` function checks the local storage for an existing cart ID.
+
+    If an existing cart ID does not exist, the function dispatches a `CREATE_GUEST_CART` action, using data from a POST request to the `/V1/guest-carts` REST endpoint, to assign a cart ID to the current session.
+
+    After the function obtains an existing cart ID, it dispatches an `ADD_ITEM_TO_CART` action, using data from a POST request to `V1/guest-carts/<cartID>/items`, to update the cart.
+    This request also updates the cart information stored on the server.
+3. When the `ADD_ITEM_TO_CART` returns with a successful response, the function calls `getCartDetails` and `toggleCart`.
+
+    The `getCartDetails` action creator makes two simultaneous calls to the `/V1/guest-carts/<cartId>` and `/V1/guest-carts/<cardId>/totals` REST endpoints.
```

Comment:
Typo in the second URI here? It says `cardId`, but is perhaps supposed to be `cartId`?